src/chrome/content/rules/Webstaurant_Store.xml

```diff
@@ -1,18 +1,21 @@
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://webstaurantstore.com/ => https://webstaurantstore.com/: Cycle detected - URL already encountered: https://www.webstaurantstore.com/
-Fetch error: http://www.webstaurantstore.com/ => https://www.webstaurantstore.com/: Cycle detected - URL already encountered: https://www.webstaurantstore.com/
-	Some pages redirect to http.
+
+	Problematic hosts in *webstaurantstore.com:
+
+		- cdnimg (different content)
+		- cdnimg1 (different content)
+		- cdnimg2 (different content)
+		- cdnimg3 (different content)
```

Comment:
These domains look good to me, such as https://cdnimg1.webstaurantstore.com/uploads/design/2014/1/webstaurantstore-logo.png
Can you give some links if there are any issues?

Reply:
The implicit test URL gives 403 over https and 301 over http, and I'm not sure how to disable it.

Reply:
You can add an exclusion for the implicit URLs, try 
`<exclusion pattern="^http://cdnimg\d?\.webstaurantstore\.com/$"/>`